frontend/src/routes/error-page.tsx

```diff
@@ -0,0 +1,20 @@
+import { isRouteErrorResponse, useRouteError } from "react-router-dom";
+
+export default function ErrorPage() {
+    const error = useRouteError() as Error;
+    console.error(error);
+
+    if (!isRouteErrorResponse(error)) {
+        return null;
+    }
+
+    return (
+        <div id="error-page">
+            <h1>Oops!</h1>
```

Comment:
Very nice! We should put a GIF from The Office right next to "Oops!" to show we are hip and in-touch, just like that company who gave me one after I applied to them!
![image](https://i.pinimg.com/originals/a7/b9/ec/a7b9eca631101d2561401a61afa6e2df.gif)

Reply:
never comment this ever again